config/webpack.config.js

```diff
@@ -41,9 +54,10 @@ var config = {
 
     // must match config.webpack.output_dir
     path: path.join(__dirname, '..', 'public', 'webpack'),
-    publicPath: '/webpack/',
+    publicPath: publicPath,
 
-    filename: production ? '[name]-[chunkhash].js' : '[name].js'
+    filename: production ? '[name]-[chunkhash].js' : '[name].js',
+    chunkFilename: production ? '[name]-[chunkhash].js' : '[name].js'
```

Comment:
what does this do?

Reply:
https://webpack.js.org/configuration/output/#output-chunkfilename
Name format for non-entry chunks created by webpack.

Reply:
The dynamic chunks didn't have `webpackChunkName` configured. I'll fix it.